src/main/daml/Daml/Finance/Interface/Instrument/Swap/Fpml/FpmlTypes.daml

```diff
@@ -366,6 +402,33 @@ data FloatingRateCalculation = FloatingRateCalculation with
     --   that is subtracted from the floating rate).
     deriving (Eq, Show)
 
+-- | A type defining a rounding direction and precision to be used in the rounding of a rate.
+data Rounding = Rounding with
```

Comment:
maybe intend as follows

`data Rounding = Rounding 
  with
    roundingDirection : RoundingDirectionEnum
      -- ^ Specifies the rounding direction.
    precision : Int
      -- ^ Specifies the rounding precision in terms of a number  of decimal places. Note how a
      -- percentage rate rounding of 5 decimal places is expressed as a rounding precision of 7 in
      -- the FpML document since the percentage is expressed as a decimal, e.g. 9.876543% (or
      -- 0.09876543) being rounded to the nearest 5 decimal places is 9.87654% (or 0.0987654).
  deriving (Eq, Show)`

Reply:
*indent